core/lens/lens_option.ml

```diff
@@ -0,0 +1,45 @@
+type 'a t = 'a option
```

Comment:
I think most, if not all, of these functions are available in `Utility`. If any of them aren't, then I suggest you implement them in `OptionUtils` and subsequently delete this module.

Reply:
See above comment regarding `List` module.